### PR TITLE
Checkbox: VR Implementation

### PR DESCRIPTION
**Pull request by eniomoura**
#### What changed?

Implemented VR changes to Checkbox

VR Off Light Mode:
image

VR Off Light Mode Focus:
image
image
image

VR Off Dark Mode:
image

VR Off Dark Mode Focus:
image
image
image

VR On Light Mode:
image

VR On Light Mode Focus:
image
image
image
image

VR On Light Mode Pressed/Hover:
image
image
image
image
image
image
image
image

VR On Dark Mode:
image

VR On Dark Mode Focus:
image
image
image
image

VR On Dark Mode Pressed/Hover:
image
image
image
image
image
image
image
image

#### Why?

To enable Web Visual Refresh.

### Links

- [Jira](https://jira.pinadmin.com/browse/CPW-7080)
- [Figma](https://www.figma.com/design/vjhfBsOtHw0wVg67vqwz1v/Gestalt-for-Web?node-id=2508-17907&m=dev)

### CODE REVIEW

**Comment by AlbertCarreras** [comment]
Screenshot by Dropbox Capture

**Comment by AlbertCarreras** [comment]
Can we use VR tokens? 
Screenshot by Dropbox Capture

**Comment by eniomoura** [comment]
> Is this shrink preventable? Brave Browser - Checkbox - Gestalt 2024-07-29 at 11 37 42 AM [ Brave Browser - Checkbox - Gestalt 2024-07-29 at 11 37 42 AM ](https://private-user-images.githubusercontent.com/10593890/353136798-b58c56d8-34c1-4015-9427-3378b9e46314.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNzA0NjUsIm5iZiI6MTcyMjI3MDE2NSwicGF0aCI6Ii8xMDU5Mzg5MC8zNTMxMzY3OTgtYjU4YzU2ZDgtMzRjMS00MDE1LTk0MjctMzM3OGI5ZTQ2MzE0LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDE2MjI0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmMjM4OWNlOTE2ODgzMThiZTRkNjE2OTgwY2I4Yzk4ZTAyMThlZTJmYWE5MGExODljZjAwNGRhY2U4ODM4ODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.G41emCytxmbGUl-gTqCHwto9bGLJSVcSo8UrzZPCtSQ) [ ](https://private-user-images.githubusercontent.com/10593890/353136798-b58c56d8-34c1-4015-9427-3378b9e46314.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNzA0NjUsIm5iZiI6MTcyMjI3MDE2NSwicGF0aCI6Ii8xMDU5Mzg5MC8zNTMxMzY3OTgtYjU4YzU2ZDgtMzRjMS00MDE1LTk0MjctMzM3OGI5ZTQ2MzE0LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDE2MjI0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmMjM4OWNlOTE2ODgzMThiZTRkNjE2OTgwY2I4Yzk4ZTAyMThlZTJmYWE5MGExODljZjAwNGRhY2U4ODM4ODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.G41emCytxmbGUl-gTqCHwto9bGLJSVcSo8UrzZPCtSQ)

Recording 2024-07-29 at 13 22 32
That's part of the design,  it seems

**Comment by eniomoura** [comment]
> Double check spacing here: Screenshot by Dropbox Capture Screenshot by Dropbox Capture Screenshot by Dropbox Capture

Fixed

File: packages/gestalt/src/Checkbox/InternalCheckbox.css [hunk 1]
```diff
@@ -59,10 +112,22 @@
   composes: flex from "../Layout.css";
   composes: xsItemsCenter from "../Layout.css";
   composes: justifyCenter from "../Layout.css";
-  composes: sizeLg from "../Borders.css";
```
[the remark this answers is not in the document]
Comment (eniomoura) [reply packages/gestalt/src/Checkbox/InternalCheckbox.css:62]: Done!

**Comment by AlbertCarreras** [comment]
Brave Browser - Checkbox - Gestalt 2024-07-29 at 3 34 46 PM

**Comment by AlbertCarreras** [comment]
Screenshot by Dropbox Capture

this might be unrelated to your PR but could you fix it?

**Review by AlbertCarreras — APPROVED** [review]

**Pull request merged**